### PR TITLE
Support multiple runners

### PR DESCRIPTION
**Pull request by nashif**

### CODE REVIEW

File: scripts/test_reports/report.py [hunk 1]
```diff
@@ -343,44 +363,407 @@ def discover(self):
...
+
+            result = r.get('result')
+
+            if result == 'PASS':
+                test_status = self.status.PASSED
+            elif result == 'SKIPPED':
+                test_status = self.status.SKIPPED
+            elif result == 'FAIL':
+                test_status = self.status.FAILED
+            elif result == 'FAIL':
```
Comment (tedd-an) [inline scripts/test_reports/report.py]: Duplicate.

**Comment by tedd-an** [comment]
The script only works with Python3 so consider to change the interpreter to Python3 at the beginning of the script. 
`#!/usr/bin/env python3`

**Comment by tedd-an** [comment]
At least for autopts, LGTM.

**Review by tedd-an — APPROVED** [review]
I only tested AutoPTS parts and

File: scripts/test_reports/report.py [hunk 2]
```diff
@@ -343,44 +369,405 @@ def discover(self):
...
 
 def main():
     args = parse_args()
 
-    # Discover TCF Testuls
-    tr = TCF(result_directory=args.junit_dir, project_id=args.project, suite=args.suite, version=args.commit, milestone=args.milestone)
+
+    if args.runner == 'maxwell':
+        tr = MaxwellPro(results_file=args.results_file,
+                         config=args.config,
```
Comment (rmstoi) [inline scripts/test_reports/report.py:738]: One extra space in indent: continuation line over-indented for visual indent

The same applies to indentation of all argument to MaxwellPro

File: scripts/test_reports/report.py [hunk 3]
```diff
@@ -343,44 +369,405 @@ def discover(self):
...
+    parser.add_argument('-c', '--config', default=None,
+            help="Configuration name.")
+
+    parser.add_argument('-V', '--commit', default=None,
+        help="Version being tested (git desribe string)")
 
     parser.add_argument('-p', '--project', type=int, help="Project ID")
     parser.add_argument('-s', '--suite', type=int, help="Suite ID")
+
     parser.add_argument('-n', '--dry-run', action="store_true", help="Dry run")
```
Comment (rmstoi) [inline scripts/test_reports/report.py]: It seem --dry-run argument is not used. It still uploads test results with this --dry-run specified.

**Pull request merged**